src/esp/sensor/VisualSensor.h

```diff
@@ -19,11 +20,24 @@ class RenderTarget;
 
 namespace sensor {
 
+using Mn::Math::Literals::operator""_degf;
+
+struct VisualSensorSpec : public SensorSpec {
+  float ortho_scale = 0.1f;
+  vec2i resolution = {84, 84};          // height x width
```

Comment:
Why 84, 84 default? Seems like an odd choice to not pick a power of 2.

Reply:
There's real no reason for this besides that this was the default value previously. Happy to change to 128, 128